fiftyone/core/dataset.py

```
@@ -1105,6 +1108,193 @@ def deleted(self):
         """Whether the dataset is deleted."""
         return self._deleted
 
+    @property
+    def active_label_schemas(self):
+        """The list of active fields in the dataset's
+        :ref:`label schemas <annotation-label-schema>`.
+        """
+        return list(self._doc.active_label_schemas)
+
+    @active_label_schemas.setter
+    def active_label_schemas(self, fields):
```

Ruff (0.14.8)

Missing return type annotation for private function `active_label_schemas`

Add return type annotation: `None`

(ANN202)